prbench/src/prbench/envs/dynamic3d/mujoco_utils.py

```diff
@@ -168,7 +174,7 @@ def set_joint_pos_quat(
 
         joint_qpos_addr = self.sim.model.get_joint_qpos_addr(name)
 
-        self.sim.data.qpos[joint_qpos_addr : joint_qpos_addr + 7] = np.array(
+        self.sim.data._data.qpos[joint_qpos_addr : joint_qpos_addr + 7] = np.array(
```

Comment:
are these changes fixing a bug? or why is this needed?

Reply:
made both "_model" and "_data" global in this [commit](https://github.com/Princeton-Robot-Planning-and-Learning/prpl-mono/pull/166/commits/44c92397d5f6aee8878ef768762248580664ad75) to avoid so many pylint disables